cmd/apiserver-boot/boot/run/local_aggregated.go

```diff
@@ -5,7 +5,7 @@ Licensed under the Apache License, Version 2.0 (the "License");
 you may not use this file except in compliance with the License.
 You may obtain a copy of the License at
 
-    http://www.apache.org/licenses/LICENSE-2.0
+	http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
minimal change

Reply:
Ah, sorry. That's my editor cleaning up whitespace. I've reverted that now.